test/helpers/kubectl.go

```diff
@@ -1404,6 +1412,35 @@ func (kub *Kubectl) ciliumInstallHelm(filename string, options map[string]string
 	return nil
 }
 
+func (kub *Kubectl) waitToDeleteCilium() error {
+	var (
+		pods []string
+		err  error
+	)
+
+	ctx, cancel := context.WithTimeout(context.Background(), HelperTimeout)
+	defer cancel()
+
+	status := 1
+	for status > 0 {
+
+		select {
+		case <-ctx.Done():
+			return fmt.Errorf("timed out waiting to delete Cilium: pods still remaining: %s", pods)
+		default:
+		}
+
+		pods, err = kub.GetCiliumPodsContext(ctx, GetCiliumNamespace(GetCurrentIntegration()))
```

Comment:
We should be able to use `CiliumNamespace` here

Reply:
Why?